src/gmail.js

```diff
@@ -961,6 +961,26 @@ var Gmail = function(localJQuery) {
             if (pathname && (pathname.endsWith("/i/s") || pathname.endsWith("/i/fd"))) {
                 api.tools.parse_request_payload(params, triggered);
             }
+            if (pathname && (pathname.endsWith("/i/s"))) {
+                if (params.body_params && params.body_params[2] && params.body_params[2][1] && params.body_params[2][1][0][2][2][7]) {
+                    // check params length  and get thread ids
+                    // console.log('params.body_params: ', params.body_params);
+                    if (params.body_params[2][1][0][2][2][7][2].length > 3) {
+                        const newThreadIds = params.body_params[2][1].map(obj => obj[2][1]);
+                        const newMessageIds = params.body_params[2][1].map(obj => obj[2][2][7][3]);
```

Comment:
What format are these IDs? Are they the new format as provided by new Gmail? Will these be compatible with the rest of our data-APIs which (for now) only supports old-style IDs?

If not... We need to make it clear somehow that the IDs provided with these events cannot be used with the rest of our APIs. Through documentation, or something.

If possible we should try to also locate IDs which are compatible with our old-style data-APIs, until they have been changed to support the new XHR data-formats.